app/components/articleSearch/components/searchItem/index.tsx

```diff
@@ -63,7 +67,28 @@ const SearchItem = (props: ISearchItemProps) => {
     currentUser,
     deleteComment,
   } = props;
-  const { title, venue, authors, year, fosList, citedCount, referenceCount, doi, id, abstract, comments } = props.paper;
+  const {
+    title,
+    venue,
+    authors,
+    year,
+    fosList,
+    citedCount,
+    referenceCount,
+    doi,
+    id,
+    abstract,
+    comments,
+    urls,
+  } = props.paper;
+
+  const pdfSourceRecord = urls.find((paperSource: IPaperSourceRecord) => {
+    if (paperSource.url.includes("pdf")) return true;
```

Comment:
I think ".pdf" will be better because there could be the url has `world'pdf'mania.com`